.github/workflows/release-generate-ci.yaml

```diff
@@ -430,12 +437,12 @@ jobs:
           GH_TOKEN: ${{ secrets.SERVERLESS_QE_ROBOT }}
           GITHUB_TOKEN: ${{ secrets.SERVERLESS_QE_ROBOT }}
         if: ${{ (github.event_name == 'push' || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule') && github.ref_name == 'main' }}
-        name: '[kn-plugin-func - release-v1.15] Create Konflux PR'
+        name: '[kn-plugin-event - release-1.15] Create Konflux PR'
         run: |
           set -x
-          repo="kn-plugin-func"
-          branch="sync-konflux-release-v1.15"
-          target_branch="release-v1.15"
+          repo="kn-plugin-event"
+          branch="sync-konflux-release-1.15"
+          target_branch="release-1.15"
```

Comment:
This looks weird. The func config shouldn't be replaced by the event one :confused:.

Reply:
It looks like it was fixed in 2b5bdc51dcd3c954290f1f88772fa4ae33d4c6a4

Something must have been confused while resolving the conflict.